docs/README.md

```diff
@@ -0,0 +1,44 @@
+# Development
+
+This is a collection of topics related to development of the SDK.
+
+## Transcoding
+
+One of the goals of this sdk is to decouple the encoding from the transport as
+much as possible. This is done by the integrating interacting with the
```

Comment:
"integrating interacting" reads odd. Perhaps reword?

Reply:
Thanks! yeah, I was trying to say sdk integrator